src/cloud-api-adaptor/podvm/files/etc/systemd/system/confidential-data-hub.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Confidential Data Hub TTRPC API Server
+After=network.target
+
+[Service]
+Type=simple
+ExecStart=/usr/local/bin/confidential-data-hub -c /run/confidential-containers/cdh.toml
```

Comment:
@mkulke is cdh.toml provided via process-user-data ?

Reply:
yes (but this has been the case prior to this pr)

Reply:
Earlier cdh was started by kata-agent and kata-agent had an explicit dep on process-user-data. Now that cdh is a separate service, should this unit be explicitly started after process-user-data?

Reply:
or the attestation-agent socket monitor is enough?

Reply:
yeah, it's implicitly required since AA requires process-user-data, but yes, it doesn't hurt to add it explicitly I guess.